browser/components/TableApprovalCell.tsx

```diff
@@ -11,35 +11,27 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-import { Cell } from 'fixed-data-table';
+
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 
 interface Props {
-  rowIndex: number;
-  field: any;
-  data: any;
-  col: any;
-  columnKey: string;
+  id: string;
 }
 
 export default class TableApprovalCell extends React.Component<
   Partial<Props>,
   {}
 > {
   render() {
-    const { rowIndex, field, data, col, columnKey, ...props } = this.props;
-    const id = data[rowIndex].contribution_id;
     return (
-      <Cell {...props}>
-        <div className="row">
-          <div className="col-md-6">
-            <Link className="btn btn-info" to={`/approvals/${id}`}>
-              Approve/Deny
-            </Link>
-          </div>
+      <div className="row center">
+        <div className="col-md-6">
+          <Link className="btn-info" to={`/approvals/${this.props.id}`}>
```

Comment:
This ends up looking strange in the table. Don't worry about fixing it now, I can help clean that up later.